src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx

```diff
@@ -27,7 +27,7 @@ Here we'll explain how New Relic users get access to specific capabilities and s
 When it comes to what New Relic features your users can access, there are two main factors: 
 
 * A user's **[user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type)**: For organizations on [our usage-based pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), your users' user type is a billing factor. The user type is what sets the maximum allowed capabilities a user can access. It overrides all role-related access. It's meant to be a fairly long-term setting based on someone's expected New Relic duties.
-* A user's assigned **roles**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**, which are the granular New Relic abilities (for example, the ability to modify New Relic APM settings). Roles are assigned by applying them to a [user group](#groups).
+* A user's assigned **roles**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**, which are the granular New Relic abilities (for example, the ability to modify New Relic APM settings). Roles are assigned by applying them to a [user group](#groups). 
```

Comment:
Role or roles? Is it possible for a user to have more than one role?

Reply:
yeah they can have multiple, hence why we have to clarify they're additive.